wg-meetings/2020-04-08.md

```diff
@@ -0,0 +1,91 @@
+# Node.js  Diagnostics WorkGroup Meeting 2020-04-08
+
+## Links
+
+* **Recording**: Unavailable
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/374
+* **Minutes Google Doc**: https://docs.google.com/document/d/1hD30I0WsXxOrDDp-2RUk1Bb4T5lsMArwYoAwA_021lc/edit
+
+## Present
+
+* Harshitha K P (@harshithakp)
+* Diagnostics team: @nodejs/diagnostics
```

Comment:
Looks like this is missing some people who were ther?

Reply:
looks like we missed to register our attendance. I added those now based on what I recollect,  so team - if this is incorrect, feel free to propose edits here (actually I joined a bit late due to overlap with tsc, and left early)